core/domain/user_jobs_one_off_test.py

```diff
@@ -171,6 +219,9 @@ def _run_one_off_job(self):
             self.count_jobs_in_taskqueue(
                 taskqueue_services.QUEUE_NAME_ONE_OFF_JOBS), 1)
         self.process_and_flush_pending_tasks()
+        # Test reduce function does nothing.
```

Comment:
In the future if you come across this, could you just get rid of the reduce function and test running the job manually?

If it works then you don't need this at all. In general, you shouldn't be calling reduce() separately though.

Reply:
I tested it manually. It works! Removed reduce.